_includes/gs/windows_gs.html

```diff
@@ -17,6 +17,15 @@
           <p>dotnet new</p>
         </div>
       </div>
+      <p>or a sample F# Hello World application!</p>
+      <div class="terminal">
+        <div class="terminal-titlebar"></div>
+        <div class="terminal-body">
+            <p>mkdir hwapp</p>
+            <p>cd hwapp</p>
+            <p>dotnet new --lang f#</p>
```

Comment:
Is it possible to use "fsharp" here too?  If so may be better to use that here since "#" often needs escaping